src/components/photoGallery/PhotoGallery.scss

```
@@ -0,0 +1,14 @@
+.imageGallery {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-around;
+    gap: 10px;
+}
+
+.spaceImage {
+    width: 200px;
+    height: 200px;
+    object-fit: cover;
+    
+}
```

This ⛔ means there isn't a newline at the end of the file! Standard practise is for the files to contain _exactly one_ completely blank (that includes spaces) line at the end of every file.
```suggestion
}
```